src/templates/remove-exclusive-edit-mode-modal.html

```diff
@@ -0,0 +1,26 @@
+{# Modal for confirming the removal of exclusive edit mode when enforced #}
+{% if mode == 'edit' %}
+  {% set label = 'You have enabled "Enforce exclusive edit mode". Unlocking this entry will redirect you to view mode. If you enter edit mode, exclusive edit will be reactivated.'|trans %}
+{% else %}
+  {% set label = 'You have enabled "Enforce exclusive edit mode". If you enter edit mode, exclusive edit will be reactivated.'|trans %}
+{% endif %}
+<div class='modal fade' id='removeExclusiveEditModal' tabindex='-1' role='dialog' aria-labelledby='removeExclusiveEditLabel'>
+  <div class='modal-dialog' role='document'>
+    <div class='modal-content'>
+      <div class='modal-header'>
+        <h5 class='modal-title' id='removeExclusiveEditLabel'>{{ 'Remove exclusive edit mode for this entry'|trans }}</h5>
+        <button type='button' class='close' data-dismiss='modal' aria-label='{{ 'Close'|trans }}'>
+          <span aria-hidden='true'>&times;</span>
+        </button>
+      </div>
+      <div class='modal-body'>
+        <label>{{ label }}</label>
+        <label>{{ 'You can change this setting in your %sprofile%s.'|trans|format("<a href='ucp.php'>", "</a>")|raw }}</label>
+      </div>
+      <div class='modal-footer'>
+        <button type='button' class='btn btn-secondary' data-dismiss='modal'>{{ 'Close'|trans }}</button>
+        <button type='button' class='btn btn-primary' data-action='remove-exclusive-edit' data-dismiss='modal'>{{ 'Confirm removal'|trans }}</button>
```

Comment:
just use "Confirm" so we can reuse the translated string in other places

Reply:
Alright
I checked and since we haven't used confirm, I figured let's go with the action itself
![image](https://github.com/user-attachments/assets/19087562-36c5-49f4-8de6-5f0e76b23e2e)
```suggestion
{% set action = entityRequestActionsArr|first %}
{% if action and action.target_userid == App.Users.userData.userid %}
  <button type='button' class='btn btn-primary' data-action='remove-exclusive-edit' data-dismiss='modal'>
    {{ action.action }}
  </button>
{% endif %}
```

Reply:
Went with "Confirm". Thank you